tests/scripts/testcases_prepare.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euxo pipefail
 
-: ${ANSIBLE_MAJOR_VERSION:=2.10}
+: ${ANSIBLE_MAJOR_VERSION:=2.12}
```

Comment:
After merging this part into a release version, we can remove requirements-2.10.txt.

Reply:
We already have a few breaking changes, in kubespray 2.1.
How about we just up the default version to 2.12 in kubespray 2.19 and drop 2.9 and 2.10 in kubespray 2.20?

Reply:
That is a nice idea, thanks @cristicalin 
OK, let's change the default version to 2.12 with keeping current ansible version tests.
Then we can drop 2.9 and 2.10 after releasing 2.19.